src/main/java/io/inisos/bank4j/util/Iso20022ReferenceElementValidator.java

```diff
@@ -0,0 +1,71 @@
+package io.inisos.bank4j.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+public class Iso20022ReferenceElementValidator {
+    private static final Pattern ISO20022_VALID_CHARS_PATTERN = Pattern.compile("^[A-Za-z0-9\\(\\)\\+\\,\\-\\./:\\s]+$");
```

Comment:
IntellIj warns me about redundant escapes, are they intended?

Reply:
No, they were added automatically by copying and pasting. Fixed now, thanks.